imod_coupler/drivers/dfm_metamod/config.py

```diff
@@ -65,6 +72,18 @@ def validate_mf6_msw_well_pkg(
     def resolve_mf6_msw_node_map(cls, mf6_msw_node_map: FilePath) -> FilePath:
         return mf6_msw_node_map.resolve()
 
+    @validator(
+        "mf6_river_to_dfm_1d_q_dmm",
+        "dfm_1d_waterlevel_to_mf6_river_stage_dmm",
+        "mf6_river2_to_dmf_1d_q_dmm",
+        "mf6_drainage_to_dfm_1d_q_dmm",
+        "msw_runoff_to_dfm_1d_q_dmm",
+        "dfm_2d_waterlevels_to_msw_h_dmm",
+        "dfm_1d_points_dat",
+    )
+    def resolve_mapping_file(cls, mapping_file: FilePath) -> FilePath:
```

Comment:
```suggestion
    def resolve_mapping_files(cls, mapping_file: FilePath) -> FilePath:
```

Reply:
I didn't know you could add multiple members to these validators.
Can you do the same to the mapping files in metamod for consistency?

Reply:
done